2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /home/hugo/TFG/Notebooks/wandb/run-20250604_141343-5ver7wuk/logs/debug.log
2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /home/hugo/TFG/Notebooks/wandb/run-20250604_141343-5ver7wuk/logs/debug-internal.log
2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:monkeypatch_ipython():643] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x73d8495c8230>
2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:init():852] calling init triggers
2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'batch_size': 256, 'learning_rate': 0.001, 'hidden_size': 98, 'epochs': 80, '_wandb': {}}
2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:init():893] starting backend
2025-06-04 14:13:43,501 INFO    MainThread:6235 [wandb_init.py:init():897] sending inform_init request
2025-06-04 14:13:43,503 INFO    MainThread:6235 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-06-04 14:13:43,503 INFO    MainThread:6235 [wandb_init.py:init():907] backend started and connected
2025-06-04 14:13:43,509 INFO    MainThread:6235 [wandb_run.py:_label_probe_notebook():1369] probe notebook
2025-06-04 14:13:43,510 INFO    MainThread:6235 [wandb_init.py:init():1002] updated telemetry
2025-06-04 14:13:43,518 INFO    MainThread:6235 [wandb_init.py:init():1026] communicating run to backend with 90.0 second timeout
2025-06-04 14:13:44,111 INFO    MainThread:6235 [wandb_init.py:init():1101] starting run threads in backend
2025-06-04 14:13:44,552 INFO    MainThread:6235 [wandb_run.py:_console_start():2566] atexit reg
2025-06-04 14:13:44,553 INFO    MainThread:6235 [wandb_run.py:_redirect():2414] redirect: wrap_raw
2025-06-04 14:13:44,553 INFO    MainThread:6235 [wandb_run.py:_redirect():2483] Wrapping output streams.
2025-06-04 14:13:44,553 INFO    MainThread:6235 [wandb_run.py:_redirect():2506] Redirects installed.
2025-06-04 14:13:44,554 INFO    MainThread:6235 [wandb_init.py:init():1147] run started, returning control to user process
2025-06-04 14:14:35,084 INFO    MainThread:6235 [wandb_run.py:_config_callback():1429] config_cb ('_wandb', 'visualize', 'test_confusion_matrix') {'panel_type': 'Vega2', 'panel_config': {'panelDefId': 'wandb/confusion_matrix/v1', 'fieldSettings': {'Actual': 'Actual', 'Predicted': 'Predicted', 'nPredictions': 'nPredictions'}, 'stringSettings': {'title': 'Confusion Matrix Curve'}, 'transform': {'name': 'tableWithLeafColNames'}, 'userQuery': {'queryFields': [{'name': 'runSets', 'args': [{'name': 'runSets', 'value': '${runSets}'}], 'fields': [{'name': 'id', 'fields': []}, {'name': 'name', 'fields': []}, {'name': '_defaultColorIndex', 'fields': []}, {'name': 'summaryTable', 'args': [{'name': 'tableKey', 'value': 'test_confusion_matrix_table'}], 'fields': []}]}]}}} None
2025-06-04 14:14:35,332 INFO    MainThread:6235 [wandb_run.py:_finish():2314] finishing run TFG_Hugo/TFG_MUL_EVAL/5ver7wuk
2025-06-04 14:14:35,332 INFO    MainThread:6235 [jupyter.py:save_history():477] not saving jupyter history
2025-06-04 14:14:35,332 INFO    MainThread:6235 [jupyter.py:save_ipynb():401] not saving jupyter notebook
2025-06-04 14:14:35,332 INFO    MainThread:6235 [wandb_init.py:_jupyter_teardown():628] cleaning up jupyter logic
2025-06-04 14:14:35,332 INFO    MainThread:6235 [wandb_run.py:_atexit_cleanup():2531] got exitcode: 0
2025-06-04 14:14:35,333 INFO    MainThread:6235 [wandb_run.py:_restore():2513] restore
2025-06-04 14:14:35,333 INFO    MainThread:6235 [wandb_run.py:_restore():2519] restore done
2025-06-04 14:14:38,707 INFO    MainThread:6235 [wandb_run.py:_footer_history_summary_info():4160] rendering history
2025-06-04 14:14:38,707 INFO    MainThread:6235 [wandb_run.py:_footer_history_summary_info():4192] rendering summary
2025-06-04 14:14:38,709 INFO    MainThread:6235 [wandb_run.py:_footer_sync_info():4121] logging synced files
